# HG changeset patch
# User Dmitri A. Sergatskov <dasergatskov@gmail.com>
# Date 1352725783 18000
#      Mon Nov 12 08:09:43 2012 -0500
# Branch stable
# Node ID 773892abfd93493e880d32d6a7038e326065368b
# Parent  7ba5ff95467e3524622118d50eff0bee32c7e350
Add correct typo ghostscript_binary -> ghostscript.binary.

* scripts/plot/private/__print_parse_opts__.m: Correct typo.
ghostscript_binary -> ghostscript.binary ("." not "_").

diff --git a/scripts/plot/private/__print_parse_opts__.m b/scripts/plot/private/__print_parse_opts__.m
--- a/scripts/plot/private/__print_parse_opts__.m
+++ b/scripts/plot/private/__print_parse_opts__.m
@@ -118,17 +118,17 @@ function arg_st = __print_parse_opts__ (
       elseif (strncmp (arg, "-PSTOEDIT:", 9))
         arg_st.pstoedit_binary = arg{10:end};
       elseif ((length (arg) > 2) && arg(1:2) == "-G")
         arg_st.ghostscript.binary = file_in_path (getenv ("PATH"), arg(3:end));
         if (isempty (arg_st.ghostscript.binary))
           error ("print: Ghostscript binary ""%s"" could not be located",
                  arg(3:end));
         else
-          arg_st.ghostscript_binary = __quote_path__ (arg_st.ghostscript_binary);
+          arg_st.ghostscript.binary = __quote_path__ (arg_st.ghostscript.binary);
         endif
       elseif (length (arg) > 2 && arg(1:2) == "-F")
         idx = rindex (arg, ":");
         if (idx)
           arg_st.font = arg(3:idx-1);
           arg_st.fontsize = str2num (arg(idx+1:end));
         else
           arg_st.font = arg(3:end);
